client/src/components/Tickets/TicketModal.jsx

```diff
@@ -166,17 +166,18 @@ function TicketHistoryReply({ ticketId }) {
                 <Message type="error">Something went wrong.</Message>
               )}
               <Button
-                type="submit"
-                color="primary"
+                attr="submit"
+                type="primary"
                 disabled={
                   !isValid ||
                   isSubmitting ||
                   isReplying ||
                   gettingTicketHistory ||
                   loadingError
                 }
+                isLoading={isReplying}
               >
-                {isReplying && <Spinner size="sm" color="white" />} Reply
+                Reply
```

Comment:
Looks fine. Modal does not close after submit. Should it? Beyond this PR I know. Reminder to self.

<details>

I don't even know what should happen. I made a [complex decision document](https://confluence.tacc.utexas.edu/pages/viewpage.action?pageId=120455500) that I can't even follow. Wes fail.

</details>

Reply:
Consider myself reminded. Resolving comment.